pennylane/qinfo/transforms.py

```diff
@@ -462,7 +466,7 @@ def circ(params):
             [2.16840434e-18, 2.81967252e-01]]]))
 
     """
-    new_qnode = _make_probs(qnode, post_processing_fn=lambda x: qml.math.squeeze(qml.math.stack(x)))
+    new_qnode = _make_probs(qnode)
```

Comment:
Oh so it was also not necessary in the old return system case?

Reply:
If `post_processing_fn` is not passed, it defaults to `lambda x: qml.math.squeeze(qml.math.stack(x))`.

Reply:
Ok thanks!